### ISSUE TITLE
Not able to reproduce results for FB15k-237

### ISSUE THREAD

**Issue by Nilesh-Agrawal** [ibalazevic/HypER#3]
Can you please let know the hyperparameters you used for the results mentioned in paper.

When running with default hypermarater, I was not able to reproduce results.
Please find mrr metric value below, with default hyperparameter.

300
0.0010591120226308703
Validation:
Number of data points: 35070
Hits @10: 0.5104362703165098
Hits @3: 0.3603364699173082
Hits @1: 0.24342743085258056
Mean rank: 383.916880524665
Mean reciprocal rank: 0.33071295056716465
Test:
Number of data points: 40932
Hits @10: 0.5053259063813154
Hits @3: 0.3573487735756865
Hits @1: 0.23783347991791265
Mean rank: 394.60920551158017
Mean reciprocal rank: 0.32585497990251716

Please find below hyperparameters used for FB15K-237:
experiment = Experiment(model_name, num_iterations=300, batch_size=128, learning_rate=0.001,
                            decay_rate=0.99, ent_vec_dim=200, rel_vec_dim=200, cuda=True,
                            input_dropout=0.3, hidden_dropout=0.3, feature_map_dropout=0.2,
                            in_channels=1, out_channels=32, filt_h=1, filt_w=9, label_smoothing=0.1)

**Comment by ibalazevic** [comment]
Hi, I just added a table with hyperparameter values for each dataset to the README. Hope this helps.

**Comment by Nilesh-Agrawal** [comment]
Thanks a lot.
I was able to reproduce results.
